packages/web/utils/dework.ts

```diff
@@ -61,9 +77,23 @@ export const processDeworkData = (data: DeworkData) => {
     });
     // get usdc rewards
     element?.rewards?.forEach(({ token, amount }) => {
-      if (usdcTokens.includes(token.address || 'empty')) return;
-      if (amount) {
-        totalEarnedInUSDC += +amount;
+      if (
+        usdcTokens6Decimals.includes(token.address?.toLowerCase() || 'empty') &&
+        amount
+      ) {
+        totalEarnedInUSDC += +amount / 10 ** 6;
```

Comment:
I know `+var` will coerce a variable, but it's using a side effect that some people might not know whereas `Number(var)` shows it's a cast.